src/test/groovy/com/palantir/gradle/versions/ConsistentVersionsPluginIntegrationSpec.groovy

```diff
@@ -41,24 +41,32 @@ class ConsistentVersionsPluginIntegrationSpec extends IntegrationSpec {
         makePlatformPom(mavenRepo, "org", "platform", "1.0")
 
         buildFile << """
-            buildscript {
-                repositories {
-                    mavenCentral()
-                }
-                dependencies {
-                    classpath 'com.palantir.configurationresolver:gradle-configuration-resolver-plugin:0.3.0'
-                }
-            }
             plugins {
                 id '${PLUGIN_NAME}'
             }
             allprojects {
-                apply plugin: 'com.palantir.configuration-resolver'
+                tasks.register("resolveConfigurations", {
```

Comment:
I haven't looked too deeply into why, but on Gradle 7, we fail to resolve the `gradle-configuration-resolver-plugin`. However given that the [plugin is archived](https://github.com/palantir/gradle-configuration-resolver-plugin), I pulled the relevant task out into directly into this test and removed the dependency.